### DISCUSSION TITLE
请问<input type="checkbox">如何实现禁止修改，又不会像添加disabled属性那样变灰？

### DISCUSSION THREAD

**Discussion by SouSuoKe** [layui/layui#1935] [问答] [answered]
![Snipaste_2024-05-22_15-35-41-574](https://github.com/layui/layui/assets/27963195/fffcc123-d85b-4f2a-9248-ff52cde2d210)

如图，展示页面上使用`<input type="checkbox">`显示了相关设置，但是不想让用户在展示页面修改这个设置，使用disabled又会使checkbox变灰没有那么好看，请问如何设置能使checkbox样式不变又不能修改？

**Comment by Sight-wcg** [comment, marked as answer]
如果是想修改表格中的 checkbox，可以参考这个示例 https://stackblitz.com/edit/x61zk4?file=index.html

如果是非表格中的 checkbox，可以参考文档中 checkbox 标签的 lay-skin 自定义属性，通过 lay-sikin 自定义 checkbox 皮肤，修改方法和上面的类似。

**Reply by SouSuoKe** [reply]
感谢大佬，没想到直接css解决了，一直在研究复选框事件。。。
